packages/spor-accordion-react-native/src/Expandable.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react";
+import {
+  composeRestyleFunctions,
+  createVariant,
+  SpacingProps,
+  spacingShorthand,
+  SpacingShorthandProps,
+  useRestyle,
+  useTheme,
+  VariantProps,
+} from "@shopify/restyle";
+import React from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { Text } from "@vygruppen/spor-typography-react-native";
+import { Pressable } from "react-native";
+import {
+  DropdownDownFill18Icon,
+  DropdownUpFill18Icon,
+  DropdownUpFill24Icon,
+  DropdownDownFill24Icon,
+  DropdownDownFill30Icon,
+  DropdownUpFill30Icon,
+} from "@vygruppen/spor-icon-react-native";
+import { ExpandableItem } from "./ExpandableItem";
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  VariantProps<Theme, "expandableVariant", "variant">;
+const variant = createVariant({ themeKey: "expandableVariant" });
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacingShorthand,
+  variant,
+]);
+
+type ExpandableVariant = "card" | "outline" | "list";
+type ExpandableSize = "sm" | "md" | "lg";
+
+type ExpandableProps = Exclude<RestyleProps, "variant"> & {
+  title: string;
+  children: React.ReactNode;
+  leftIcon?: JSX.Element;
+  variant: ExpandableVariant;
+  isInitiallyExpanded?: boolean;
```

Comment:
Jeg har sett et pattern der man kaller denne typen prop for "defaultWhatever", så i dette tilfellet "defaultExpanded". Høres det ut som en god idé? Er litt på gjerdet selv.